doc/plugins.dox

```diff
@@ -329,24 +422,30 @@ Now we need to clean up properly when the plugin is unloaded.
 To remove the menu item from the Tools menu, you can use gtk_widget_destroy().
 gtk_widget_destroy() expects a pointer to a GtkWidget object.
 
-First you should add gtk_widget_destroy() to your plugin_cleanup() function.
-The argument for gtk_widget_destroy() is the widget object you created earlier in
-plugin_init(). To be able to access this pointer in plugin_cleanup(), you need to move
-its definition from plugin_init() into the global context so its visibility will increase
+First you should add gtk_widget_destroy() to your @ref GeanyPluginFuncs::cleanup() function. The
+argument for gtk_widget_destroy() is the widget object you created earlier in @ref
+GeanyPluginFuncs::init(). To be able to access this pointer in GeanyPluginFuncs::cleanup(), you
+need to use geany_plugin_set_data() to set plugin-defined data pointer to the widget.
+Alternatively, you can store the pointer in some global variable so its visibility will increase
```

Comment:
Maybe mention one can wrap several data in a struct passed to `geany_plugin_set_data()`?